config/software/jemalloc.rb

```diff
@@ -17,8 +17,8 @@
   source :md5 => md5,
          :url => "http://src.fedoraproject.org/repo/pkgs/jemalloc/jemalloc-#{version}.tar.bz2/#{md5}/jemalloc-#{version}.tar.bz2"
 end
-version("5.0.1") do
```

Comment:
[Optional] It's ok to leave the version in there, but we will have to switch to another version anyway when they fix the issue, so it's ok to remove it as well.

Reply:
I thought about leaving it here. But it's a broken version... Even when we upgrade in the future, it's going to be a different version. So I ended up getting rid of it.